locales/en.json

```diff
@@ -140,14 +141,14 @@
 				"invalidYear": "This expiry year is not a valid"
 			},
 			"addressLine1": {
-				"name": "building name/number and street",
-				"message": "Enter a valid billing address",
-				"containsTooManyDigits": "Enter a valid billing address"
+				"name": "address line 1",
+				"message": "Enter address line 1, typically the building and street",
+				"containsTooManyDigits": "Enter address line 1, typically the building and street"
 			},
 			"addressLine2": {
-				"name": "additional address line",
-				"message": "Enter valid address information",
-				"containsTooManyDigits": "Enter valid address information"
+				"name": "address line 2",
+				"message": "Enter address line 2",
+				"containsTooManyDigits": "Enter address line 2"
```

Comment:
I’m a bit concerned that this does not really tell people what the problem is (too many digits). It might trip people up entering foreign addresses. For example, in Germany, the postcode is written immediately before the city so it’s conceivable that someone could put it in the address line 2 input because that’s the one before the city… and German postcodes are all digits (but only five digits, so not enough to trigger our card number check). But since this is an existing issue I’m not going to hold up the review on this.

Reply:
Good point.  I didn't realise German postcodes work like that.
We noticed a few minor existing bugs with the validation and separate tickets for them.